src/wire-protocol.ts

```diff
@@ -132,7 +132,6 @@ export const isQueryResponse = (res: any): res is QueryResponse<any> =>
   isQueryResponse(res) || isQueryFailure(res);
 
 /**
-<<<<<<< HEAD
```

Comment:
:D

Reply:
yea, snuck that in XD. I don't when it showed up, it caused tests to fail 🤷🏻